niri-ipc/src/state.rs

```diff
@@ -63,6 +66,13 @@ pub struct KeyboardLayoutsState {
     pub keyboard_layouts: Option<KeyboardLayouts>,
 }
 
+/// The overview state communicated over the event stream.
+#[derive(Debug, Default)]
+pub struct OverviewState {
+    /// State of the overview
+    pub is_open: bool,
+}
+
 impl EventStreamStatePart for EventStreamState {
     fn replicate(&self) -> Vec<Event> {
         let mut events = Vec::new();
```

Comment:
Forgot to add it here and below. Which means that if you start event-stream when the overview is open, it won't be notified as such.

Reply:
I think I resolved this? Let me know if it looks good.